Tests/UnitTests/APIToRequestMappingTests.swift

```diff
@@ -213,4 +225,11 @@ class APIToRequestMappingTests: XCTestCase {
         }
         XCTAssert(try session.size(window: "myWindow") == (width: 500, height: 500))
     }
+
+    func testMaximizeSelectedWindow() throws {
```

Comment:
```suggestion
    func testWindowMaximize() throws {
```
For consistency with the selenium request url.